quickjs.h

```diff
@@ -676,13 +676,20 @@ JS_BOOL JS_SetConstructorBit(JSContext *ctx, JSValueConst func_obj, JS_BOOL val)
 JSValue JS_NewArray(JSContext *ctx);
 int JS_IsArray(JSContext *ctx, JSValueConst val);
 
+typedef struct InlineCache InlineCache;
```

Comment:
Should be properly namespaced:
```suggestion
typedef struct JSInlineCache JSInlineCache;
```